### ISSUE TITLE
BUG - cannot change ports in port forwarding

### ISSUE THREAD

**Issue by Eldenroot** [gzenux/asuswrt-rtn18u#36]
![image](https://user-images.githubusercontent.com/778890/98484976-748c3200-2213-11eb-8bbf-87d0c1953e0b.png)

I cannot type into the inpu boxes here...

**Comment by Eldenroot** [comment]
Cannot write ":" or ",", just numbers

**Comment by gzenux** [comment]
I tried to reproduce this issue in 384.18 and got the following results:
1. Work fine for "External Port" field => Cannot reproduce.
2. Only numbers are accepted in "Internal Port" field => The result is the same as your description.

I suppose you are talking aobut "Internal Port" field for this issue. It seems to be not a bug or might be a known issue in my opinion since the input data validation rule of this field and the backend firewall rule command are matched (The traffic will be forwarded to specified internal port of the specified internal IP).

It doesn't support internal port range mapping in the current backend code. I will not try to change the backend code since I think it's better to update/fix the design from upstream.

**Comment by Eldenroot** [comment]
Nope, it happens in external port. You can write all symbols but when you wanna edit again, it is impossible to do that...

**Comment by gzenux** [comment]
I still cannot reproduce on it. Both ',' and ':' can be used if I clear this field (I find that only one ':' can be used when I edit an existing record).
Are there any specific steps or conditions to reproduce this issue?

**Comment by Eldenroot** [comment]
How to use more :?

Also how can I set port forwarding (same ports) for two different mac/ip?

**Comment by gzenux** [comment]
1. Use (1) two entries to achieve that or (2) copy & paste
2. https://superuser.com/questions/874002/can-you-port-forward-the-same-port-to-2-different-internal-ip-addresses
    It's recommended to use two different ports for the different services
